code/datums/actions/mobs/create_turrets.dm

```diff
@@ -0,0 +1,114 @@
+/datum/action/cooldown/mob_cooldown/create_turrets
+	name = "Create Sentinels"
+	button_icon = 'icons/mob/simple/lavaland/lavaland_monsters.dmi'
+	button_icon_state = "legion_turret"
+	desc = "Create legion sentinels that fire at any enemies."
+	cooldown_time = 2 SECONDS
+	var/minimum_turrets = 2
+	var/maximum_turrets = 2
```

Comment:
As in other classes, document these vars and procs. An explanation as to why we seem to be picking a random number between 2 and 2 would be a good thing to add (I can see why from examining another file, but shouldn't have to).